tools/install_mac_deps.sh

```diff
@@ -8,9 +8,19 @@ fi
 
 $BREW update
 $BREW install autoconf automake bdw-gc bison boost ccache cmake git \
-      libtool openssl pkg-config protobuf python
+      libtool openssl pkg-config python
 $BREW install gmp --c++11
 
+# Install specific version of Protobuf, since newer versions break compatibility
```

Comment:
Is it possible to just use brew like this?
https://stackoverflow.com/questions/3987683/homebrew-install-specific-version-of-formula

Reply:
It may work but as far as I can tell there is no guarantee that the required version will be available.